pkgs/tools/networking/redfang/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitLab, bluez }:
+
+stdenv.mkDerivation rec {
+  pname = "redfang";
+  version = "2.5";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = pname;
+    rev = "upstream/${version}";
+    sha256 = "sha256-dF9QmBckyHAZ+JbLr0jTmp0eMu947unJqjrTMsJAfIE=";
+  };
+
+  patches = [ ./makefile-install.diff ];
```

Comment:
Can you try to push that upstream?

I doubt that there will be a new release but other distributions may need the patch as well.

Reply:
Submitted a merge request here: https://gitlab.com/kalilinux/packages/redfang/-/merge_requests/1